src/App.tsx

```diff
@@ -126,10 +126,21 @@ const App = () => {
                     errorElement={<ErrorPage />}
                   />
                   <Route
-                    path="/production/:productionId/line/:lineId"
-                    element={<ProductionLine />}
+                    path="/production-calls/production/:productionId/line/:lineId"
+                    element={<CallsPage />}
                     errorElement={<ErrorPage />}
                   />
+                  {/* <Route
+                    path="/production-calls"
+                    element={<CallsPage />}
+                    errorElement={<ErrorPage />}
+                  >
+                    <Route
+                      path="/production-calls/production/:productionId/line/:lineId"
+                      element={<CallsPage />}
+                      errorElement={<ErrorPage />}
+                    />
+                  </Route> */}
```

Comment:
Is this commented section needed? If not in use --> remove

Reply:
@malmen237
[test: updated code to allow multiple calls on mobile](https://github.com/Eyevinn/intercom-frontend/pull/225/commits/d2d0f298d624c6cc50b110bdf96a3ef4ab157b08) is only to test if it worked on mobile